api_definition.go

```diff
@@ -205,8 +205,8 @@ func (a *APIDefinitionLoader) MakeSpec(appConfig *apidef.APIDefinition) *APISpec
 		}
 	}
 
-	newAppSpec.RxPaths = make(map[string][]URLSpec)
-	newAppSpec.WhiteListEnabled = make(map[string]bool)
+	newAppSpec.RxPaths = make(map[string][]URLSpec, len(appConfig.VersionData.Versions))
```

Comment:
It's common practice to do it if the amount of inserts on a slice or map is known at compile-time. You can argue that if it's not a hot path it's not worth the time, but in this case the time is already invested :)